test/controllers/api/user_blocks_controller_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+module Api
+  class UserBlocksControllerTest < ActionDispatch::IntegrationTest
+    def test_routes
+      assert_routing(
+        { :path => "/api/0.6/user_blocks/1", :method => :get },
+        { :controller => "api/user_blocks", :action => "show", :id => "1" }
+      )
+      assert_recognizes(
+        { :controller => "api/user_blocks", :action => "show", :id => "1", :format => "xml" },
+        { :path => "/api/0.6/user_blocks/1.xml", :method => :get }
+      )
```

Comment:
We don't normally make the XML case explicit like that though I guess maybe we should have implicit and explicit cases for it but looking at other API controller tests at the moment only the JSON ones are explicit.

Reply:
I copied it from some other place where it was.

Reply:
This is where I copied it from:

https://github.com/openstreetmap/openstreetmap-website/blob/0e21afc56593eb9fce14d8cfe0984494e758c150/test/controllers/api/notes_controller_test.rb#L23

Reply:
Most of assert_recognizes tests were added for paths differing from the default ones
https://github.com/openstreetmap/openstreetmap-website/commit/90e46a58de583ac2bd7bb43077faa94186957a7a

I guess it's not required here, so I removed it. And added JSON.